app/stats.go

```diff
@@ -5,10 +5,10 @@ import (
 	"github.com/codegangsta/cli"
 )
 
-func LsStats() cli.Command {
+func LsJournal() cli.Command {
 	return cli.Command{
-		Name:      "ls-stats",
-		ShortName: "stats",
+		Name:      "ls-journal",
```

Comment:
I can't think of any other operation we will have for journal now. I suppose flush can be done automatically with you call `journal`. Unlike replica, or snapshot, we're not going to remove/add journal etc. Should be fine to remove it.